src/styles/blocks/calendar.scss

```
@@ -0,0 +1,46 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: $gapBetweenCell;
+  padding: 10px;
+  width: $width;
+
+  &__day {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+
+    height: $cellSize;
+    width: $cellSize;
+    border: 1px solid #000;
+    background-color: #eee;
+    cursor: pointer;
+    transition-duration: 0.5s;
+    // transition: transform 0.5s;
```

```suggestion
    transition-property: transform, background-color;
```
default value of transition propery is 'all' which is not good for performance